docs/sources/panels/configure-thresholds/_index.md

```diff
@@ -0,0 +1,102 @@
+weight: 300
+aliases:
+  [
+    docs/grafana/latest/panels/thresholds/,
+    docs/grafana/latest/panels/,
+    docs/grafana/latest/panels/specify-thresholds/about-thresholds/,
+    docs/grafana/latest/panels/specify-thresholds/add-a-threshold/,
+    docs/grafana/latest/panels/specify-thresholds/add-threshold-to-graph/,
+    docs/grafana/latest/panels/specify-thresholds/delete-a-threshold/,
+  ]
+---
+
+# Configure visualization thresholds
+
+This section includes information about using thresholds in your visualizations. You'll learn about thresholds and their defaults, how to add or delete a threshold, and adding a threshold to a legacy panel.
+
+## About thresholds
+
+A threshold is a value that you specify for a metric that is visually reflected in a dashboard when the threshold value is met or exceeded.
+
+Thresholds provide one method for you to conditionally style and color your visualizations based on query results. You can apply thresholds to most, but not all, visualizations. For more information about visualizations, refer to [Visualization panels]({{< relref "../../visualizations" >}}).
+
+You can use thresholds to:
+
+- Color lines or areas in the [Graph panel]({{< relref "../../visualizations/graph-panel" >}})
+- Color lines in the [Time-series panel]({{< relref "../../visualizations/time-series/graph-color-scheme/#from-thresholds" >}})
+- Color the background or value text in the [Stat panel]({{< relref "../../visualizations/stat-panel" >}})
+- Color the gauge and threshold markers in the [Gauge panel]({{< relref "../../visualizations/gauge-panel" >}})
+- Color markers in the [Geomap panel]({{< relref "../../visualizations/geomap" >}})
+- Color cell text or background in the [Table panel]({{< relref "../../visualizations/table" >}})
+- Define regions and region colors in the [State timeline panel]({{< relref "../../visualizations/state-timeline" >}})
+
+There are two types of thresholds:
+
+- **Absolute** thresholds are defined by a number. For example, 80 on a scale of 1 to 150.
+- **Percentage** thresholds are defined relative to minimum or maximum. For example, 80 percent.
+
+### Default thresholds
+
+On visualizations that support it, Grafana sets default threshold values of:
+
+- 80 = red
+- Base = green
+- Mode = Absolute
+
+The **Base** value represents minus infinity. It is generally the “good” color.
+
+## Add or delete a threshold
+
+You can add as many thresholds to a panel as you want. Grafana automatically sorts thresholds values from highest to lowest.
+
+Delete a threshold when it is no longer relevant to your business operations. When you delete a threshold, the system removes the threshold from all visualizations that include the threshold.
+
+**Before you begin:**
+
+- [Add a panel to a dashboard]({{< relref "../working-with-panels/add-panel.md" >}})
+
+**To add or delete a threshold**:
+
+1. To add a threshold:
+
+   a. Edit the panel to which you want to add a threshold.
```

Comment:
Here it also says you need a panel, (see comment above). This step is good, but makes the comment section of adding a panel unnecessary.

Reply:
corrected